Testscripts/Windows/LIS-INSTALL-SCENARIOS.ps1

```diff
@@ -33,7 +33,7 @@ Function Upgrade-LIS ($LISTarballUrlOld, $LISTarballUrlCurrent, $allVMData , $Te
         $CurrentLISExtractCommand = "rm -rf LISISO^wget $($LISTarballUrlCurrent)^tar -xzf $($LISTarballUrlCurrent | Split-Path -Leaf)^cp -ar LISISO/* ."
         $LISExtractCommands = $CurrentLISExtractCommand.Split("^")
         foreach ( $LISExtractCommand in $LISExtractCommands ) {
-            Run-LinuxCmd -username "root" -password $password -ip $allVMData.PublicIP -port $allVMData.SSHPort -command $LISExtractCommand
+            Run-LinuxCmd -username "root" -password $password -ip $allVMData.PublicIP -port $allVMData.SSHPort -command $LISExtractCommand -runMaxAllowedTime 20000
```

Comment:
the value is too high, that's over 5 hours for the extract command? please fix all 20.000 occurrences

Reply:
changed to -runMaxAllowedTime 2000 -maxRetryCount 3